K2Bridge/Models/Request/Queries/Query.cs

```diff
@@ -19,6 +19,12 @@ internal class Query : KustoQLBase, IVisitable
         [JsonProperty("bool")]
         public BoolQuery Bool { get; set; }
 
+        /// <summary>
+        /// Gets or sets the id with which to perform a ViewSingleDocument query.
+        /// </summary>
+        [JsonProperty("ids")]
+        public DocumentIds DocumentId { get; set; }
```

Comment:
I'm not sure, but perhaps the property name should follow the type?

Reply:
I prefer the code will be clearer, but understand you point. Changed, thanks